lib/modules/onc_program_procedure.yml

```diff
@@ -0,0 +1,1219 @@
+          though we do test for all resources which implies that we require all scopes.
+        inferno_tests:
+          - SPB-OSLS-01 - SPB-OSLS-08
+          - EHB-OELS-01 - EHB-OELS-10
+          - AVU-01 - AVU-12
+          - USCP-08
+          - USCAI-03
+          - USCCP-05
+          - USCCT-02
+          - USCC-06
+          - USCID-03
+          - USCDRRN-07
+          - USCDRLRR-07
+          - USCDR-08
+          - USCG-04
+          - USCI-04
+          - USCMR-05
+          - USCSSO-06
+          - USCPWHO-06
+          - USCLRO-06
+          - USCPBMIAO-06
+          - USCPO-06
+          - USCOBH-06
+          - USCOBT-06
+          - USCOBP-06
+          - USCOBW-06
+          - USCOHC-06
+          - USCOHR-06
+          - USCORR-06
+          - USCPROC-05
+          - USCL-01
+          - USCO-01
+          - USCPR-01
+          - USCPRO-01
+          - USCPROV-01
+      - index: 10 
+        SUT: |
+          [Both] The health IT developer demonstrates the ability of the
+          Health IT Module to deny an application’s authorization request
+          according to a patient’s preferences selected in step 9 of this
+          section in accordance with the implementation specification
+          adopted in § 170.215(a)(3).
+        TLV: |
+          [Both] The tester verifiesthe ability of the Health IT Module to
+          deny an application’s authorization request according to a
+          patient’s preferences selected in step 9 of this section in
+          accordance with the implementation specification adopted in §
+          170.215(a)(3).
+        inferno_supported: partial
+        inferno_notes: |
+          The tests currently do not check for the ability to deny second order
+          (non-USCDI) resources  because our tests are not
+          sophisticated enough in this area to find references to check.
+        inferno_tests:
+          - SPB-OSLS-01 - SPB-OSLS-08
+          - AVR-01 - AVR-15
+      - index: 11
+        SUT: |
+          [Both] The health IT developer demonstrates the ability of Health IT Module to
+          evaluate the authorization request and request end-user input, if
+          applicable (required for patient-facing applications), including the
+          ability for the end-user to explicitly enable the “offline_access”
+          scope according to the implementation specification adopted in §
+          170.215(a)(3).
+        TLV: |
+          [Both] The tester verifies the ability of Health IT Module to
+          evaluate the authorization request and request end-user input, if
+          applicable (required for patient-facing applications), including the
+          ability for the end-user to explicitly enable the “offline_access”
+          scope according to the implementation specification adopted in §
+          170.215(a)(3).
+        inferno_supported: partial
+        inferno_notes: |
```

Comment:
This isn't clear in my shorthand here, but I meant it as "Tests 01 through 08".  I stopped at 08 because that is where we know that the authorization request actually worked completely, and you can check the scope returned.  I was a little loose here on the mapping, to be more precise, these are the steps that need to pass:

```
- SPB-OSLS-02
- SPB-OSLS-03
- SPB-OSLS-07
- SPB-OSLS-08
- SPB-OSLS-10
```